build.gradle

```diff
@@ -164,6 +164,7 @@ dependencies {
     implementation "org.apache.commons:commons-csv:1.10.0"
     zipArchive group: 'org.opensearch.plugin', name:'opensearch-job-scheduler', version: "${opensearch_build}"
     compileOnly "org.opensearch:opensearch-job-scheduler-spi:${opensearch_build}"
+    implementation "com.github.seancfoley:ipaddress:5.4.0"
```

Comment:
Not sure if we should use this library. This is maintained by a user and not under any project. Lets refrain from using it.

Reply:
Please check this PR. The library is being used in multiple OS plugins already. https://github.com/opensearch-project/sql/pull/2042

Reply:
https://github.com/opensearch-project/geospatial/issues/574